core/spec/lib/spree/core/number_generator_spec.rb

```diff
@@ -0,0 +1,173 @@
+require 'spec_helper'
+
+describe Spree::Core::NumberGenerator do
+  let(:number_generator) { described_class.new(options) }
+  let(:random)           { instance_double(Random)      }
+
+  let(:model) do
+    options = options().freeze
```

Comment:
Useless assignment to variable - `options`.<br>Do not use parentheses for method calls with no arguments.